test_regress/t/t_preproc_resolve.py

```diff
@@ -16,7 +16,7 @@
 test.compile(
     # Override default flags
     v_flags=[''],
-    verilator_flags=["-E -P --preproc-resolve -y t/t_preproc_resolve"],
+    verilator_flags=["-E -P --preproc-resolve verilator_config.vlt -y t/t_preproc_resolve"],
```

Comment:
```suggestion
    verilator_flags=["-E -P --preproc-resolve t/t_preproc_resolve_config.vlt -y t/t_preproc_resolve"],
```

Avoid making a new subdirectory for a single file.

Reply:
This folder already existed (it is used by `--preproc-resolve` test). Nevertheless, I moved this config file out of the folder.